include/openPMD/backend/Attributable.hpp

```diff
@@ -203,6 +203,8 @@ class AttributableImpl
     internal::SeriesInternal const & retrieveSeries() const;
     internal::SeriesInternal & retrieveSeries();
 
+    std::vector< std::string > myPath() const;
```

Comment:
Can you please add a doxygen string for this one? :)

Reply:
I think there are also some other places where a little comment will help, so yep will do

Reply:
Done